### PR TITLE
Docs for 3.7

### PR DESCRIPTION
**Pull request by lukasbestle**
- [x] Router hooks:
  https://github.com/getkirby/kirby/pull/4260
  https://github.com/getkirby/kirby/pull/4290
- [ ] Replaced Panel screenshots throughout the docs https://github.com/getkirby/kirby/pull/4330 (@bastianallgeier)
   - [x] Home page
   - [x] Presskit
   - [ ] Guide
   - [ ] Reference
- [x] New Environment class https://github.com/getkirby/kirby/pull/4285 (@lukasbestle)
- [x] Fine-tuned cache handling based on request data https://github.com/getkirby/kirby/pull/4277 (@lukasbestle)
- [x] Find methods https://github.com/getkirby/kirby/pull/4357 (@lukasbestle)
- [x] New `k-table` component: https://github.com/getkirby/kirby/pull/4215
- [x] New `$helpers.object.isEmpty(value)` method: https://github.com/getkirby/kirby/pull/4215 (@distantnative)
- [x] New stats section: https://github.com/getkirby/kirby/pull/4298 (@bastianallgeier)
- [x] Section search: https://github.com/getkirby/kirby/pull/4316
- [x] Global helper functions: https://github.com/getkirby/kirby/pull/4018 (@distantnative)
- [x] New Toggles field: https://github.com/getkirby/kirby/pull/4343 (@bastianallgeier) 
- [x] New `layout: table` option https://github.com/getkirby/kirby/pull/4328 (@bastianallgeier)
   - [x] pages section
   - [x] files section
- [x] Target pages for security health checks: `https://getkirby.com/security/***` (@bastianallgeier)
- [x] New mobile option to keep cells in responsive views: https://github.com/getkirby/kirby/pull/4405
   - [x] structure field (@afbora)
   - [x] pages section (@afbora)
   - [x] files section (@afbora)
- [x] Removed builder and editor field migration for blocks field https://github.com/getkirby/kirby/pull/4326  (@afbora)

###  For release page
- [x] Deprecated blueprint `presets`
- [x] Add breaking change about `dump` and `e` helpers. Because of we removed the `function_exists` check.  (@afbora)

### On release
- [ ] Before merging docs PR, create a branch from main for archived 3.6 docs
- [ ] Upload 3.6 archived docs to subdomain
- [ ] Upload new `ui.json`

### CODE REVIEW

File: content/docs/3_reference/3_panel/4_sections/0_pages/reference-panelsection.txt [hunk 1]
```diff
@@ -194,6 +194,18 @@ columns:
     value: "{{ page.date.toDate('d.m.Y') }}"
 ```
 
+##### Mobile
+
+Set the `mobile` option to `true` for a column to visible in the mobile view.
```
Comment (texnixe) [inline content/docs/3_reference/3_panel/4_sections/0_pages/reference-panelsection.txt]: Set the `mobile` option to `true` for a column to be visible in the mobile view.

File: content/docs/3_reference/3_panel/3_fields/0_structure/reference-article.txt [hunk 2]
```diff
@@ -166,6 +167,31 @@ fields:
         text: this is some long text
 ```
 
+### Set columns to visible on mobile
+
+`mobile` option keeps cells in responsive views. If none of the columns are set to `mobile`, only the first column will be visible on mobile.
```
Comment (texnixe) [inline content/docs/3_reference/3_panel/3_fields/0_structure/reference-article.txt]: The `mobile` option …

File: content/docs/3_reference/3_panel/4_sections/0_pages/reference-panelsection.txt (hunk 1 above)
Comment (afbora) [reply content/docs/3_reference/3_panel/4_sections/0_pages/reference-panelsection.txt]: Thank you. You can correct my typos directly 🙂

File: content/docs/3_reference/2_templates/0_helpers/reference-helpers.txt [hunk 3]
```diff
@@ -4,3 +4,30 @@ Title: Helpers
...
+</warning>
+
+To deactivate one of Kirby's helper functions globally, you need to set a specific constant in your `index.php` to `true`, e.g.
+
+```php "/index.php"
+<?php
+
+require __DIR__ . '/kirby/bootstrap.php';
+
+define('KIRBY_HELPER_DUMP', false);
```
Comment (afbora) [inline content/docs/3_reference/2_templates/0_helpers/reference-helpers.txt]: `define` should be first before autoload:

````php
define('KIRBY_HELPER_DUMP', false);

require __DIR__ . '/kirby/bootstrap.php';

File: content/docs/3_reference/2_templates/0_helpers/reference-helpers.txt [hunk 4]
```diff
@@ -4,3 +4,30 @@ Title: Helpers
...
+
+echo (new Kirby)->render();
+```
+
+The name of the corresponding constant is always `KIRBY_HELPER` + name of the helper (uppercased). When you disable a helper function, it won't be registered anymore by Kirby.
```
Comment (afbora) [inline content/docs/3_reference/2_templates/0_helpers/reference-helpers.txt]: `KIRBY_HELPER` should be `KIRBY_HELPER_`

**Comment by distantnative** [comment]
I don't think the router hooks need any extra docs than the automatically generated ones, do they?

**Comment by bastianallgeier** [comment]
I had a look yesterday briefly and also couldn't figure out a place where to put it. Same with the Environment class actually

**Comment by distantnative** [comment]
I think the only docs the new environment class needs are checking if the config docs are still correct for setting up domains/ports etc

**Comment by afbora** [comment]
Some api endpoints that are deprecated seem not to be updated. Is it normal? 

For ex: `/api/site/title` 
http://getkirby.test/docs/reference/api/site/change-title

Related content file:
https://github.com/getkirby/getkirby.com/blob/release/3.7.0/content/docs/3_reference/8_api/0_site/15_change-title/reference-endpoint.txt

**Comment by distantnative** [comment]
@afbora not sure what you mean. `api/site/title` isn't deprecated, is it?

**Comment by afbora** [comment]
@distantnative Sorry, you're right. I mixed with JS API Methods.

**Pull request merged**